app/models/course/condition.rb

```diff
@@ -5,4 +5,6 @@ class Course::Condition < ActiveRecord::Base
   belongs_to :conditional, polymorphic: true
 
   delegate :satisfied_by?, to: :actable
+
+  NAMES = ['achievement', 'assessment', 'level']
```

Comment:
Can call the class method `Course::Conditions.condition_types` or something to that effect